src/scripts/templates/unit_activity.html

```diff
@@ -1,5 +1,14 @@
 <div class="panel panel-default">
     <div class="panel-heading">
+        <h2>Profile</h2>
```

Comment:
That was a general idea - it was required by many people, so I wanted to make it work for now and fix it later, when we have time